i18n/countries.php

```diff
@@ -139,7 +140,7 @@
 	'LI' => __( 'Liechtenstein', 'woocommerce' ),
 	'LT' => __( 'Lithuania', 'woocommerce' ),
 	'LU' => __( 'Luxembourg', 'woocommerce' ),
-	'MO' => __( 'Macao S.A.R., China', 'woocommerce' ),
+	'MO' => __( 'Macau', 'woocommerce' ),
```

Comment:
I believe the short name recommended by CLDR is `Macao` and not `Macau`: https://github.com/unicode-org/cldr/blob/release-35-1/common/main/en.xml#L1046

Am I missing something?

Reply:
We might want to use the same spelling here and in: https://github.com/woocommerce/woocommerce/blob/master/i18n/states.php#L288 (related #24555).

Reply:
Funny that in the documentation was different, I'll update both.